arcgis-ios-sdk-samples/Layers/Group layers/README.md

```diff
@@ -10,13 +10,17 @@ Group layers communicate to the user that layers are related and can be managed
 
 ## How to use the sample
 
-The layers in the map will be displayed in a table of contents. Toggle the switch next to a layer's name to change its visibility. Turning a group layer's visibility off will override the visibility of its child layers.
+The layers in the map will be displayed in a table of contents. Toggle the switch next to a layer's name or tap the cell to change its visibility. Turning a group layer's visibility off will override the visibility of its child layers.
 
 ## How it works
 
 1. Create an `AGSGroupLayer` object.
 2. Add a child layer to the group layer's layers collection.
-3. To toggle the visibility of the group, simply change the group layer's `isVisible` property.
+3. Set the group layer's `groupVisibilityMode` to change its behavior:
+    * `groupVisibilityMode.INDEPENDENT` allows each sublayer to change its visibility independently.
+    * `groupVisibilityMode.EXCLUSIVE` allows only one sublayer to be visible at a time.
+    * `groupVisibilityMode.INHERITED` treats the group layer as if it is one merged layer.
```

Comment:
I also forget how to we annotate the enum cases.

Was it wrapping cases in backticks or bare strings? i.e. `exclusive` vs exclusive vs `.exclusive`

Wait for Phil's comment.